pcdsdevices/spectrometer.py

```diff
@@ -339,14 +349,17 @@ class TMOSpectrometer(BaseInterface, GroupDevice, LightpathMixin):
     tab_component_names = True
 
     # Motor components: can read/write positions
-    lens_x = Cpt(BeckhoffAxis, ':MMS:01', kind='normal')
-    lens_pitch_up_down = Cpt(BeckhoffAxis, ':MMS:10', kind='normal')
-    lens_yaw_left_right = Cpt(BeckhoffAxis, ':MMS:11', kind='normal')
-    foil_x = Cpt(BeckhoffAxis, ':MMS:02', kind='normal')
     zone_plate = Cpt(FZPStates, 'SP1K4:FZP:STATE', add_prefix=(), kind='normal')
     zone_plate_x = Cpt(BeckhoffAxis, ':MMS:03', kind='normal')
     zone_plate_y = Cpt(BeckhoffAxis, ':MMS:04', kind='normal')
     zone_plate_z = Cpt(BeckhoffAxis, ':MMS:05', kind='normal')
+    solid_att = Cpt(SOLIDATTStates, 'SP1K4:ATT:STATE', add_prefix=(), kind='normal')
+    # Solid_att x and Y are FOIL x and y
+    solid_att_x = Cpt(BeckhoffAxis, ':MMS:02', kind='normal')
```

Comment:
It'd be nice to have some docs on these to tell non-TMO experts more about these axes.
You can do that by the following, for example:
```python
solid_att_x = Cpt(BeckhoffAxis, ":MMS:02", doc="X-axis of solid attenuator (FOIL) which does *something*")
```